app/src/main/res/navigation/navigation.xml

```diff
@@ -92,4 +95,15 @@
         android:id="@+id/accountFragment"
         android:name="com.example.sharingang.AccountFragment"
         android:label="Account" />
+    <fragment
+        android:id="@+id/wishlistViewFragment"
+        android:name="com.example.sharingang.WishlistViewFragment"
+        android:label="WishlistViewFragment" >
```

Comment:
Change the label so the appBar doesn't say "WishListViewFragment" but just "Wishlist"
```suggestion
        android:label="Wishlist" >
```